test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGetPackageManagerTests/NuGetPackageManagerTelemetryTests.cs

```diff
@@ -179,39 +186,46 @@ public async Task PreviewInstallPackage_VersionNotInRange_RaiseTelemetryEventsWi
             var telemetryService = new NuGetVSTelemetryService(telemetrySession.Object);
             TelemetryActivity.NuGetTelemetryService = telemetryService;
 
-            // Create Package Manager
-            using var solutionManager = new TestSolutionManager();
+            var testLogger = new TestLogger();
+
+            var nuGetVersioningPackageContext = new SimpleTestPackageContext("NuGet.Versioning", "1.0.7");
+            nuGetVersioningPackageContext.AddFile("lib/uap10.0/NuGet.Versioning.dll");
+
+            await SimpleTestPackageUtility.CreateFolderFeedV3Async(pathContext.PackageSource, nuGetVersioningPackageContext);
+            var sourceRepository = CreateMockSourceRepository(pathContext.UserPackagesFolder);
+            var sourceRepositoryProvider = TestSourceRepositoryUtility.CreateSourceRepositoryProvider(sourceRepository.PackageSource);
+            var sources = sourceRepositoryProvider.GetRepositories();
+            var testSettings = TestSourceRepositoryUtility.PopulateSettingsWithSources(sourceRepositoryProvider, pathContext.WorkingDirectory);
+
             var nuGetPackageManager = new NuGetPackageManager(
                 sourceRepositoryProvider,
-                Settings.LoadSpecificSettings(solutionManager.SolutionDirectory, "NuGet.Config"),
+                NullSettings.Instance,
```

Comment:
I think you can pass testSettings.

Reply:
Good catch - don't see that it's impacting the test either way, but would rather it be hooked up!